app/routes.py

```
@@ -792,7 +831,48 @@ async def meetingbaas_webhook(request: Request):
+    across processes/threads: only one caller creates the file, the rest get
+    FileExistsError and skip. bot_id is a validated UUID (see the webhook gate),
+    so it is safe in the filename.
+    """
+    if not bot_id:
+        return False
+    try:
+        fd = os.open(
+            _summary_claim_path(bot_id), os.O_CREAT | os.O_EXCL | os.O_WRONLY, 0o600
+        )
+        os.close(fd)
+        return True
```

Ruff (0.15.21)

[warning] Consider moving this statement to an `else` block

(TRY300)